### DISCUSSION TITLE
Updates! (or lack thereof)

### DISCUSSION THREAD

**Discussion by nommiin** [nommiin/ImGui_GM#8] [General]
Howdy, I had no idea GitHub had this discussions thing but neat!

Just making this post to say that ImGui_GM is not dead (not sure if anyone thought that either lol), I've just sorta been busy!!! But I want to make sure the extension still works and stuff... so expect updates soon maybe possibly? Unless everything is flawless then like hell yeah.

Also thanks for 90+ downloads on itch and 20+ stars here, y'all are too kind!

**Comment by GlebTsereteli** [comment]
Awesome, thanks! I've been sloooowly writing my own wrapper for some core elements, gonna show you when it's (if ever) ready 😆
